### DISCUSSION TITLE
Bot Names

### DISCUSSION THREAD

**Discussion by NickFellah** [thebigsleepjoe/TTT-Bots-2#47] [General]
Bots Names i thought of are: minecraft_pro, masterminion, murderer, xx_snipergodly_xx, gas_station, toyota, teddy_bear, stop_signs_are_dumb, and Xx_fortnitegod_xX
thanks!

**Comment by thebigsleepjoe** [comment]
Hi Nick! Thanks for the suggestions.

I'm hijacking this discussion to say this:
**If anyone has name suggestions, please leave them in this thread!**

NOTE: Not _all_ community names will be accepted! Names are 100% subject to my personal discretion (no offense!), and the time it takes me to add them to the global pool depends solely on how actively I am maintaining the repo.

**Reply by DaBassman1** [reply]
thebassman43 for a name

**Comment by thebigsleepjoe** [comment]
6/12/24 in v1.3: adding names from @NickFellah and @DaBassman1:

- thebassman43
- teddy_bear
- stop_signs_are_dumb
- murderer
